dropwizard-client/src/test/java/io/dropwizard/client/HttpClientBuilderTest.java

```diff
@@ -50,10 +50,7 @@
 import org.apache.http.impl.conn.SystemDefaultRoutePlanner;
 import org.apache.http.message.BasicHeader;
 import org.apache.http.message.BasicListHeaderIterator;
-import org.apache.http.protocol.BasicHttpContext;
-import org.apache.http.protocol.HTTP;
-import org.apache.http.protocol.HttpContext;
-import org.apache.http.protocol.HttpProcessor;
+import org.apache.http.protocol.*;
```

Comment:
I know you probably didn't do this, but we don't use wildcard imports in this project :)

Reply:
ah yes it's my intellij 😅 have updated.